### PR TITLE
[RI2 Ch04] Update runbook installation on provider infrastructure

### PR DESCRIPTION
**Pull request by zhuguanyu5**
Signed-off-by: zhuguanyu <zhuguanyu5@huawei.com>

### CODE REVIEW

File: doc/ref_impl/cntt-ri2/chapters/chapter04.rst [hunk 1]
```diff
@@ -131,12 +131,12 @@ References for the above features:
 -  `Intel Device Plugins for Kubernetes <https://github.com/intel/intel-device-plugins-for-kubernetes>`__
 -  `Telemetry Aware Scheduling
    <https://github.com/intel/platform-aware-scheduling/tree/master/telemetry-aware-scheduling>`__
+-  `Latest Deployment Requirements/Specification <https://wiki.anuket.io/pages/viewpage.action?pageId=36569490>`__
 
 Additional settings are available in the BMRA templates located in ``playbooks/roles/bmra-config/templates``. Changing
 these might have unexpected results and should generally not be done.
 
-You will also have to modify environmental variables defined in ``deploy.env`` to match your setup. For deploying
-Kuberef on preprovisioned infrastructure, set ``deployment_type=k8s``.
```
Comment (xudan16) [inline doc/ref_impl/cntt-ri2/chapters/chapter04.rst:139]: why remove this?

File: doc/ref_impl/cntt-ri2/chapters/chapter04.rst [hunk 2]
```diff
@@ -175,6 +175,23 @@ follows:
      "pods": "110"
    }
 
+Installation on Provider Infrastructure
```
Comment (xudan16) [inline doc/ref_impl/cntt-ri2/chapters/chapter04.rst:178]: preprovisioned Infrastructure

File: doc/ref_impl/cntt-ri2/chapters/chapter04.rst [hunk 3]
```diff
@@ -175,6 +175,23 @@ follows:
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The main steps are the same as :ref:`chapters/chapter04:Installation on Bare Metal Infratructure`.
+
+Make sure the infrastructure preprovisioned follows ``docs/release/installation/deployment-guide.rst``.
+
+When modifying the environmental variables defined in ``deploy.env``,
+set deployment type ``DEPLOYMENT:-k8s``.
```
Comment (xudan16) [inline doc/ref_impl/cntt-ri2/chapters/chapter04.rst:186]: export DEPLOYMENT=k8s

File: doc/ref_impl/cntt-ri2/chapters/chapter04.rst [hunk 4]
```diff
@@ -175,6 +175,23 @@ follows:
...
+If you are not the root user account, you neet set ``USERNAME`` to consistent the username of VM and yours,
```
Comment (xudan16) [inline doc/ref_impl/cntt-ri2/chapters/chapter04.rst:187]: Please desrcibe how to set these params and what should be the values of them.

**Review by steelescotr — APPROVED** [review]

**Review by gkunz — APPROVED** [review]

**Review by xudan16 — APPROVED** [review]

**Review by CsatariGergely — APPROVED** [review]

File: doc/ref_impl/cntt-ri2/chapters/chapter04.rst (hunk 2 above)
Comment (zhuguanyu5) [reply doc/ref_impl/cntt-ri2/chapters/chapter04.rst]: done

File: doc/ref_impl/cntt-ri2/chapters/chapter04.rst (hunk 3 above)
Comment (zhuguanyu5) [reply doc/ref_impl/cntt-ri2/chapters/chapter04.rst]: done

File: doc/ref_impl/cntt-ri2/chapters/chapter04.rst (hunk 4 above)
Comment (zhuguanyu5) [reply doc/ref_impl/cntt-ri2/chapters/chapter04.rst]: done

**Pull request merged**